source/channels/react-to-messages.rst

```diff
@@ -4,64 +4,84 @@ React to messages using emojis
-  :alt: Select a default skin tone preference for people-based emojis.
+    **Upload custom emojis**
 
-Upload custom emojis
----------------------
-  
-Everyone in your Mattermost workspace can use the custom emojis you upload. From the Emoji Picker, select **Custom Emoji**. Small, square pictures work best when selecting an image to upload. The file can be any JPG, GIF, or PNG that's up to one MB in size.
-  
-1. Enter a name for your custom emoji. This is the name that shows up in the emoji autocomplete.
-2. Choose **Select**, then select the image to use for the emoji. 
-3. Select **Save**. Once saved, your emoji is added to the list of custom emoji.
-  
-.. image:: ../images/add_custom_emoji.png
-  
-.. note::
+     Everyone in your Mattermost workspace can use the custom emojis you upload. From the emoji picker, select **Custom Emoji**. Small, square pictures work best when selecting an image to upload. The file can be any JPG, GIF, or PNG that's up to 1 MB in size.
 
-  If you can't see the **Custom Emoji** option, then your Mattermost System Admin may have restricted access to upload custom emoji. Contact your Mattermost System Admin for assistance.
-  
-To use your custom emoji in a message, select it from the Emoji Picker, or type ``:`` followed by your emoji name to bring it up in the emoji autocomplete. 
-  
-Remove custom emojis
---------------------
-  
-You can remove custom emojis that you uploaded to Mattermost.
+     1. Enter a name for your custom emoji. This is the name that shows up in the emoji autocomplete.
+     2. Choose **Select**, then select the image to use for the emoji.
+     3. Select **Save**. Once saved, your emoji is added to the list of custom emoji.
+
+      .. image:: ../images/add_custom_emoji.png
+
+    .. note::
+
+       If you can't see the **Custom Emoji** option, then your Mattermost System Admin may have restricted access to upload custom emoji. Contact your Mattermost System Admin for assistance.
   
-1. Open the Emoji Picker.
-2. Select **Custom Emoji**.
-3. If required, use the Search Bar to find your custom emoji in the list.
-4. Under **Actions** select **Delete**.
-5. Choose **Delete** to confirm.
+     To use your custom emoji in a message, select it from the emoji picker, or type ``:`` followed by your emoji name to bring it up in the emoji autocomplete.
+
+    **Remove custom emojis**
+      
+      You can remove custom emojis that you uploaded to Mattermost.
   
-.. image:: ../images/delete_custom_emoji.png
-   :alt: Remove custom emoji.
+     1. Open the emoji picker.
+     2. Select **Custom Emoji**.
+     3. If required, use the Search bar to find your custom emoji in the list.
+     4. Under **Actions** select **Delete**.
+     5. Choose **Delete** to confirm.
+
+       .. image:: ../images/delete_custom_emoji.png
+           :alt: Remove custom emoji
+      
+  .. tab:: Mobile app
+   
+    **To react to messages with emoji in the mobile app v2**
```

Comment:
Please remove references to mobile v2 for now. We'll incorporate needed references to v2 as we get closer to v2 being released.